tests/zfs-tests/include/blkdev.shlib

```diff
@@ -595,7 +595,7 @@ function list_file_blocks # input_file
 	else
 		AWK='awk'
 	fi
-	log_must zpool sync -f
+	sync_pool $TESTPOOL true
```

Comment:
```suggestion
	sync_pool $pool true
```

Reply:
Even that is possibly not the same, since the old code did all pools